subsys/mpsl/mpsl_init.c

```diff
@@ -132,6 +132,8 @@ static int mpsl_lib_init(const struct device *dev)
 
 	clock_cfg.source = m_config_clock_source_get();
 	clock_cfg.accuracy_ppm = CONFIG_CLOCK_CONTROL_NRF_ACCURACY;
+	clock_cfg.skip_wait_lfclk_started =
```

Comment:
No need to use `IS_ENABLED` here

Reply:
My mistake. It is a choice symbol. Therefore you either need to use ifdefs or `IS_ENABLED`. So you can keep it as is